.github/workflows/ci.yml

```diff
@@ -8,27 +8,36 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-          - 2.4
-          - 2.5
-          - 2.6
+          - '2.5'
```

Comment:
Should we add a required_ruby_version to the gemspec?

Reply:
On it